engine/src/main/scala/cromwell/engine/workflow/tokens/JobTokenDispenserActor.scala

```diff
@@ -26,14 +30,31 @@ class JobTokenDispenserActor(override val serviceRegistryActor: ActorRef,
                              logInterval: Option[FiniteDuration],
                              dispenserType: String,
                              tokenAllocatedDescription: String)
-  extends Actor with ActorLogging with JobInstrumentation with CromwellInstrumentationScheduler with Timers with DynamicRateLimiter {
+  extends Actor
+    with ActorLogging
+    with JobInstrumentation
+    with CromwellInstrumentationScheduler
+    with Timers
+    with DynamicRateLimiter
+    with CromwellInstrumentation
+{
+
+  // Metrics paths are based on the dispenser type
+  private val tokenDispenserMetricsBasePath: NonEmptyList[String] = NonEmptyList.of("token_distributor", dispenserType)
```

Comment:
*cough* distributor *cough*

Reply:
Should we rename the class to `...DistributorActor` too then?